src/App.js

```diff
@@ -1,8 +1,27 @@
-import React from "react";
-import Router from "./router";
+import CardHolder from "../src/components/typography/CardHolder";
+//import React from "react";
+import React, { useState } from "react";
+import Navbar from "../src/components/typography/Navbar";
 
 function App() {
-  return <Router />;
+  const [mode, setMode] = useState("light");
+
+  const toggleMode = () => {
+    if (mode === "light") {
+      setMode("dark");
+      document.body.style.backgroundColor = "black";
+    } else {
+      setMode("light");
+      document.body.style.backgroundColor = "white";
+    }
+  };
+
+  return (
+    <>
+      <Navbar title="Home" subtitle="Dashboard" mode={mode} toggleMode={toggleMode} />
```

Comment:
check if your prettier format is working or not

Reply:
Prettier format is working fine